mysql-test/mroonga/storage/column/int/t/with_index_null_value.test

```diff
@@ -0,0 +1,42 @@
+# Copyright(C) 2024 Takuya Kodama <otegami@clear-code.com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+--source ../../../../include/mroonga/have_mroonga.inc
+--source ../../../include/mroonga/load_mroonga_functions.inc
```

Comment:
The number of `../` are different.

Reply:
fix: [08c15be](https://github.com/mroonga/mroonga/pull/790/commits/08c15bee929323a3d42e00047e2ecefc08be2a4f)